autoblocks/_impl/global_state.py

```diff
@@ -25,10 +33,31 @@ def init() -> None:
         daemon=True,
     )
     background_thread.start()
-
+    for sig in [signal.SIGINT, signal.SIGTERM]:
+        _loop.add_signal_handler(sig, lambda: asyncio.create_task(_on_exit_signal()))
     _started = True
 
 
+async def _on_exit_signal() -> None:
```

Comment:
do we need both this + the atexit handler in tracer.py?

Reply:
Removed at exit / signal handler stuff for now